chrome/content/ui.js

```diff
@@ -392,6 +393,14 @@ var ui = {
     Services.ww.openWindow(null, privDialog, "", features, args);
   },
 
+  onAccountCreated: function(acc) {
+    let args = {
+      account: acc.normalizedName,
+      protocol: acc.protocol.normalizedName,
+    };
+    ui.generate(args);
```

Comment:
`ui.generate()` is going to open a blocking modal, which is what we want to avoid.

Instead, you should directly call,

```
otr.generatePrivateKey(acc.normalizedName, acc.protocol.normalizedName)
```

which will launch the async worker.

We'll want to suppress any error here though. So,

```
otr.generatePrivateKey( ... ).catch(function(err) {
  Cu.reportError(err);
});
```